### PR TITLE
Issue/7701 strange and inconsistent sub itemization in documentation

### PR DESCRIPTION
**Pull request by Hugo-Inmanta**
# Description

closes https://github.com/inmanta/inmanta-core/issues/7701

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [x] Attached issue to pull request
- [x] Changelog entry
~~- [ ] Type annotations are present~~
~~- [ ] Code is clear and sufficiently documented~~
~~- [ ] No (preventable) type errors (check using make mypy or make mypy-diff)~~
~~- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)~~
~~- [ ] Correct, in line with design~~
~~- [ ] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )~~
~~- [ ] If this PR fixes a race condition in the test suite, also push the fix to the relevant stable branche(s) (see [test-fixes](https://internal.inmanta.com/development/core/tasks/build-master.html#test-fixes) for more info)~~

### CODE REVIEW

File: docs/administrators/operational_procedures.rst
Comment (Hugo-Inmanta) [inline docs/administrators/operational_procedures.rst:1]: In this file:

- Make sure sub items are displayed consistently
- Add links to relevant settings

Looks like this:
operational_procedures

File: docs/administrators/operational_procedures_with_lsm.rst
Comment (Hugo-Inmanta) [inline docs/administrators/operational_procedures_with_lsm.rst:1]: Same as in `operational_procedures.rst`

File: docs/administrators/support.rst
Comment (Hugo-Inmanta) [inline docs/administrators/support.rst:1]: This removes one level of unnecessary indentation:
support

File: docs/administrators/waf.rst
Comment (Hugo-Inmanta) [inline docs/administrators/waf.rst:1]: This removes one level of unnecessary indentation:
waf

File: docs/language.rst
Comment (Hugo-Inmanta) [inline docs/language.rst:1]: Indent the operations so they live in their own block. Imo it looks nicer in the context:
arithmetic_ops

**Review by arnaudsjs — APPROVED** [review]

**Comment by Hugo-Inmanta** [comment]
Merged in iso6 with https://github.com/inmanta/inmanta-core/pull/7819

**Pull request closed without merging**